### PR TITLE
Hide Instagram connect button

### LINKED ISSUE #7252: Hide Instagram connector button
Hide Instagram connector button in the 'add dataset' modal until the feature works properly.

### PR DESCRIPTION
**Pull request by MariaCheca**
Closes #7252

Hidden every reference to connect to Instagram until the dataservice works properly.

cc @xavijam @yasharora73

### CODE REVIEW

**Comment by Cartofante** [comment]
Can one of the admins verify this patch?

File: services/datasources/lib/datasources/datasources_factory.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@ def self.get_all_oauth_datasources
             Url::Dropbox::DATASOURCE_NAME,
             Url::Box::DATASOURCE_NAME,
             Url::GDrive::DATASOURCE_NAME,
-            Url::InstagramOAuth::DATASOURCE_NAME,
+            # Url::InstagramOAuth::DATASOURCE_NAME,
```
Comment (xavijam) [inline services/datasources/lib/datasources/datasources_factory.rb]: We don't need to to this, with the changes above is ok.

File: lib/assets/javascripts/cartodb/common/dialogs/create/listing/import_options.js [hunk 2]
```diff
@@ -139,22 +139,22 @@ module.exports = {
...
-      fileExtensions: [],
-      acceptSync: false,
-      showAvailableFormats: false,
-      fileAttrs: {
-        ext: false,
-        title: 'title'
-      }
-    }
-  },
+  // Instagram: {
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/dialogs/create/listing/import_options.js]: 👍

**Comment by xavijam** [comment]
Could we remove it in the new editor add-layer modal too?

File: services/datasources/lib/datasources/datasources_factory.rb (hunk 1 above)
Comment (MariaCheca) [inline services/datasources/lib/datasources/datasources_factory.rb]: This is necessary if we want (and we should, as it won't make sense if we disable the service) to hide Instagram from the "Connect to external data sources" section in the /account page:

screen shot 2016-04-28 at 11 39 55

**Comment by MariaCheca** [comment]
Sure! I'll update the changes asap :)

File: services/datasources/lib/datasources/datasources_factory.rb (hunk 1 above)
Comment (xavijam) [inline services/datasources/lib/datasources/datasources_factory.rb]: Ahhh, good point then ;), but please, assign people to review your changes.
Frontend: 👍 
Rails: ?

Comment (MariaCheca) [inline services/datasources/lib/datasources/datasources_factory.rb]: You're right!
@juanignaciosl can you please CR this change when you can? Thanks :)

Comment (juanignaciosl) [inline services/datasources/lib/datasources/datasources_factory.rb]: 👍

**Comment by MariaCheca** [comment]
✨

**Comment by xavijam** [comment]
Hey! you didn't say a word when you made the changes for the new editor :(

**Comment by MariaCheca** [comment]
@xavijam SORRY :( I thought you already reviewed it, totally my fault.

**Pull request merged**